superset/utils/pandas_postprocessing/pivot.py

```diff
@@ -100,11 +100,9 @@ def pivot(  # pylint: disable=too-many-arguments,too-many-locals
         margins_name=marginal_distribution_name,
     )
 
+    # removes additional columns from pandas issue #18030 described above
```

Comment:
In general I don't think we should place comments like this in the codebase, as over time it will cause a significant amount of clutter. People can always check `git blame` and take a look at the underlying PR if they need more context, and that's where info like this should be placed IMO (btw, the issue referenced here doesn't appear to be mentioned in the PR description). In addition, it was unclear to me what is meant by "described above" - there is no description directly above this block.

Reply:
Sure, I can remove this if you all prefer to keep the comments to a minimum. The intent was a to make it more obvious that the workaround for the pandas issue is a 2 step operation.

I'll add a ref to the original PR for better context, in case this ever gets fixed on pandas.